net/arp-scan/src/opnsense/mvc/app/controllers/OPNsense/ARPscanner/forms/general.xml

```diff
@@ -0,0 +1,16 @@
+<form>
+   <field>
+      <id>arpscanner.general.interface</id>
+      <label>Listen Interface</label>
+      <type>dropdown</type>
+      <help>Interface where to find the networks logically linked to.</help>
+   </field>
+   <field>
+      <id>arpscanner.general.networks</id>
+      <label>Network to probe</label>
+      <type>text</type>
+      <help>Multiple CIDR separated by comma: 192.168.1.0/24,172.16.31.0/24.
```

Comment:
best merge this to a single line, translation will otherwise pick up line noise "           " between the two sentences